### PR TITLE
More instrument adapter

### PR DESCRIPTION
**Pull request by oxade**

### CODE REVIEW

**Review by velvia — APPROVED** [review]
Thanks.  I think the level should probably be trace.

File: crates/sui-core/src/execution_engine.rs [hunk 1]
```diff
@@ -340,6 +340,7 @@ fn execute_transaction<S: BackingPackageStore + ParentSync + ChildObjectResolver
     (cost_summary, result)
 }
 
+#[instrument(name = "exec_engine_transfer_object", level = "debug", skip_all)]
```
Comment (oxade) [inline crates/sui-core/src/execution_engine.rs:343]: @velvia 
I believe this function should be very quick. It's entire in Rust and does not touch the VM.
Is the overhead from instrumenting too much?

Comment (velvia) [reply crates/sui-core/src/execution_engine.rs:343]: Hm, ok we can skip instrumenting this for now.

**Comment by oxade** [comment]
> Thanks. I think the level should probably be trace.

changed

**Pull request merged**